webapi/Controllers/MaintenanceController.cs

```diff
@@ -49,7 +49,7 @@ public class MaintenanceController : ControllerBase
 
         var migrationStatus = await migrationMonitor.GetCurrentStatusAsync(cancellationToken);
 
-        if (migrationStatus != ChatMigrationStatus.None)
+        if (migrationStatus == ChatMigrationStatus.Upgrading)
```

Comment:
I thought we WANTED to hang and display an error message when an upgrade is required???

Reply:
let's figure this out. imo we shouldn't block the entire chat experience if it works, and should probably show this error as a status alert..

Reply:
@crickman is our man, here. He's the one who put this in so he would know how we should react to it.

Reply:
Oh, and again, maybe we should make this a field in the ServiceInfo response to minimize the number of calls we make to ther server??

Reply:
@glahaye just had a sync with @crickman, we discovered that in local mode the backend will _always_ be in the `RequiresUpgrade` state since the ACS index will not exist. he is going to look into this further. 
we agreed that we should only show the "Migrating memory..." message when the migration is actually occurring, since you may be in a state where you need it but it isn't currently happening, in which case you should still be able to chat.

Reply:
If he's happy, I'm happy :)